jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/ArtifactUtilTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.kit.common.util;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.attribute.BasicFileAttributes;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.eclipse.jkube.kit.common.util.ArtifactUtil.LAST_MODIFIED_TIME_SAVE_FILENAME;
+
+class ArtifactUtilTest {
+
+  @TempDir
+  Path tempDir;
+
+  private File file;
```

Comment:
Naming here is confusing me,
maybe rename
file -> artifactFile
lastModifiedTime -> artifactFileLastModifiedTime
saveFile -> lastModifiedSavedFile